server/test/integration/dependentControllerTest.js

```diff
@@ -0,0 +1,33 @@
+var request = require('supertest');
+let chai = require('chai');
+
+let should = chai.should();
+var assert = chai.assert;
+
+describe('Dependent Information', function() {
+    var app;
+    beforeEach(function() {
+        app = require('../../index');
+    });
+    afterEach(function() {
+        app.close();
+    });
+    it('Returns status code 200 with no user ID passed', function(done) {
+        request(app)
+            .post('/dependents/getAllDependents')
+            .expect(200, done);
+    });
+    it('Returns status code 200 with valid dependent data using user ID 1', function(done) {
+        request(app)
+            .post('/dependents/getAllDependents')
+            .send('userID=1')
+            .expect(200, done);
+    });
+    it('Returns status code 404 with invalid route', function(done) {
+        request(app)
+            .post('/dependents/getAllDependent')
+            .send('userID=3')
+            .expect(404, done);
+    });
+
```

Comment:
Add a return after each test or remove this one please. Just to keep consistency within the file